src/views/3-Modeling/Modeling.vue

```diff
@@ -4,292 +4,224 @@
+            id="path5"
+            transform="translate(0 25)"
+          >
+            <path
+              id="ANN"
+              class="path5 other"
+              d="M210,125c.2,7.1,10.5,7.4,11,0A5.5,5.5,0,0,0,210,125Z"
+            />
+            <path
+              id="RNN"
+              class="path5 other"
+              d="M165,147c.2,7.1,10.4,7.4,10.9,0S165.4,140,165,147Z"
+            />
+            <path
+              id="RGCN"
+              class="path5 other"
+              d="M156.1,50.3c.2,7.2,10.4,7.5,10.9,0s-10.5-6.9-10.9,0"
+            />
+            <path
+              id="RGCN_2"
+              class="path5 other"
+              d="M144.9,160.7a1,1,0,0,0,2,0,1,1,0,1,0-2,0"
+            />
+            <path
+              id="RGCN_ptrn"
+              class="path5 other"
+              d="M227,214.3l-11.2,6.4-2.1,1.1-9.1,5.2-2,1.2-11.2,6.4V219l11.2,6.8V212.6l11.3,6.8V206.2Z"
+            />
+          </g>
+        </svg>
       </div>
       <div id="error-container" />
-      <div id="bees-container">
-        <div id="legend-container" />
+      <div id="bees-container" />
+      <div id="legend-container">
+        <svg
+          id="bees_legend"
+          xmlns="http://www.w3.org/2000/svg"
+          viewBox="0 0 1000 1000"
+        >
+          <g
+            id="legend-scale"
+            transform="translate(0, 700)"
+          >
+            <path
+              d="M228,643.5H807"
+              transform="translate(-28.24 -627.46)"
+              style="fill: none;stroke: white;stroke-miterlimit: 10;stroke-width: 3px"
+            />
+            <text
+              class="text-annotate"
+              transform="translate(50 27.42)"
+            >accurate</text>
+            <text
+              class="text-annotate"
+              transform="translate(795 27.42)"
+            >inaccurate</text>
+            <path
+              d="M247.5,656.5l-23-12,22-15"
+              transform="translate(-28.24 -627.46)"
+              style="fill: none;stroke: white;stroke-linecap: round;stroke-linejoin: round;stroke-width: 3px"
+            />
+            <path
+              d="M722.5,657.5l24-14-25-13"
+              transform="translate(30 -627.46)"
+              style="fill: none;stroke: white;stroke-linecap: round;stroke-linejoin: round;stroke-width: 3px"
+            />
+          </g>
+        </svg>
       </div>
     </figure>
-    <!--     all the scrolling elements -->
+    <!--     all the scrolling elements are created from modelingText.js content -->
     <article>
-      <div class="step-container text-content">
+      <div
+        v-for="text in text.methods"
+        :key="text.title"
+        class="step-container text-content"
+      >
         <div
-          id="ANN"
+          :id="text.flubber_id"
           class="step"
-          data-step="0"
         >
-          <p v-html="text.paragraph2" />
-        </div>
-      </div>
-      <div class="step-container text-content">
```

Comment:
Hmm so I think it is the removal of all of these `"step-container text-content"` divs that is tripping up my flubber code, b/c I had assigned ids to these that were then triggering the flubber animation. Are these now elsewhere?

Reply:
yeah those are now named in `modelinText.js`. That is basically everything relating to the text steps is defined there and this section loops through that and creates the same div structure for each step. I think this is working with the steps, I added an element to put them in there as `flubber_id`